docs/src/modules/components/AppContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
> I'm not sure if it will break without the flow type stripping plugin or not

[Yes, it does](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015&targets=&browsers=&builtIns=false&debug=false&code_lz=MYGwhgzhAEBiD29oFMAeAXZA7AJjASsmMOgHQDC8AtgA7xbboA8YWAngDTStsB80AbwBQQgL5A)